src/components/Details/Details.module.css

```diff
@@ -0,0 +1,103 @@
+.container {
+  position: relative;
+  width: 100%;
+  height: 85vh;
+  background: #d35443;
+  padding: 1rem;
+  border-radius: 5px;
+  max-height: 100%;
+  overflow: hidden;
+}
+
+.no_content {
+  text-align: center;
+}
+
+.no_content a {
+  background: #fb0;
+  color: #fff;
+  padding: 5px;
+  border-radius: 3px;
+}
+
+.map {
+  height: 100%;
+}
+
+.map img {
+  max-height: 100%;
+  width: 100%;
+  opacity: 0.2;
+}
+
+.weather {
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
```

Comment:
- [ ] You have used display: flex accross the components for more than 3-4 times, this is repeating yourself and falls under the category of bad practice for CSS. I would recommend you create a class display: flex and use it on your JSX whenever you need
![image](https://user-images.githubusercontent.com/62705947/225372400-6554b127-c4eb-4b3c-805d-057c1b7c551c.png)

Reply:
- create a class dFlex and give it to html tags when you need it

Reply:
I have done that 
Also, the const, it breaks my app whenever I use it

Reply:
> I have done that
> Also, the const, it breaks my app whenever I use it

Okay, don’t use const, since from what I saw its value never changed but if it breaks your app, just leave it like it is

Reply:
kk 
but will other reviewers approve it 
also I did the testing for the other components in the app test

Reply:
> kk
> but will other reviewers approve it
> also I did the testing for the other components in the app test

Yeah if it breaks your app then its good